magda-sleuther-broken-link/src/sleuther.ts

```diff
@@ -0,0 +1,441 @@
+import Registry from "@magda/typescript-common/dist/Registry";
+import { Record } from "@magda/typescript-common/dist/generated/registry/api";
+import unionToThrowable from "@magda/typescript-common/dist/util/union-to-throwable";
```

Comment:
At the risk of bike shedding, can I suggest that we always make the export name, file name, and import name all the same?  It's nice to not have to map names unnecessarily, e.g. press ctrl-p, type the name of the a symbol in one source file, and quickly end up at the definition of that symbol.